### PR TITLE
[RGTC-590] Backport Activity Finder

### PR DESCRIPTION
**Pull request by svicervlad**
## Steps for review

- [ ] Login as admin
- [ ] Open /activity-finder
- [ ] You should see, what  Activity Finder works fine with new features
Selection_093

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1919/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1919/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1919/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1919/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1919/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1919/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1919/blogWCAG2AAhtmlcs.txt

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1920/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1920/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1920/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1920/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1920/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1920/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1920/blogWCAG2AAhtmlcs.txt

**Comment by podarok** [comment]
retest this please

**Comment by podarok** [comment]
@svicervlad  - could you please gather a list of fixes and enhancements, collected in this PR?
We should have it for Open Y release notes. Thank you.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build1921/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/1921/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build1921/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build1921/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build1921/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build1921/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build1921/blogWCAG2AAhtmlcs.txt

File: modules/custom/openy_activity_finder/css/activity_finder.css [hunk 1]
```diff
@@ -791,6 +792,41 @@ and (-webkit-min-device-pixel-ratio: 2) {
...
+  font-size: 12px;
+}
+
+
+.schedules-data__row--alt .spots-availability {
+  border-radius: 3px;
+}
+
+@media (min-width: 768px){
+  .landing-content {
```
Comment (jediandrey) [inline modules/custom/openy_activity_finder/css/activity_finder.css:818]: Possible regressions, it's very common css rule, use something specific or find workaround

**Review by podarok — CHANGES_REQUESTED** [review]
Needs fix

**Comment by podarok** [comment]
@svicervlad @danylevskyi could you also extend a bit
`You should see, what Activity Finder works fine with new features`
?
This is not clear what are new features

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
This is done in a different scope
Now Activity Finder is in a decoupled repository https://github.com/ymcatwincities/openy_activity_finder

**Pull request closed without merging**